src/components/clusters/ClusterDetailsMultiRegion/components/AccessControl/ExternalAuthentication/BreakGlassCredentialList.tsx

```diff
@@ -196,7 +204,14 @@ export const BreakGlassCredentialList = ({
               <Tr key={cred.id}>
                 <Td>{cred.id}</Td>
                 <Td>{cred.username}</Td>
-                <Td>{new Date(cred.expiration_timestamp as string).toLocaleString('en-CA')}</Td>
+                <Td>
+                  <Timestamp
+                    date={new Date(cred.expiration_timestamp as string)}
+                    dateFormat={TimestampFormat.short}
+                    timeFormat={TimestampFormat.medium}
+                    locale="en-CA"
```

Comment:
Why are we hardcoding locale="en-CA"?

Reply:
I'm not a fan but this is how the local was set before (see before code).  Changing the actual date format is beyond the scope of this work - it is to just implement the use of Timestamp so date/time display so it will be easier to implement future work (which hopefully includes coming up with some type of standard)